x-pack/platform/plugins/shared/alerting/public/pages/maintenance_windows/maintenance_window_edit_page.tsx

```diff
@@ -38,22 +38,10 @@ export const MaintenanceWindowsEditPage = React.memo(() => {
   return (
     <EuiPageSection restrictWidth={true}>
       <PageHeader showBackButton={true} title={i18n.EDIT_MAINTENANCE_WINDOW} />
-      <EuiSpacer size="xl" />
-      {showMultipleSolutionsWarning && (
-        <>
-          <EuiCallOut
-            data-test-subj="maintenanceWindowMultipleSolutionsWarning"
-            title={i18n.SOLUTION_CONFIG_REMOVAL_WARNING_TITLE}
-            color="warning"
-          >
-            <p>{i18n.SOLUTION_CONFIG_REMOVAL_WARNING_SUBTITLE}</p>
-          </EuiCallOut>
-          <EuiSpacer size="xl" />
-        </>
-      )}
       <CreateMaintenanceWindowForm
         initialValue={maintenanceWindow}
         maintenanceWindowId={maintenanceWindowId}
+        showMultipleSolutionsWarning={showMultipleSolutionsWarning}
```

Comment:
I don't like this being a prop tbh, I think we can remove it from `useGetMaintenanceWindow` and everywhere else.

The decision is now to show the banner always, so why do we need this?

And even if we decide to move forward with showing the banner only if the MW has a category set, we can just use the form data for that.